developers/README.md

```diff
@@ -17,6 +17,44 @@ Supported development platforms:
 * [Ubuntu](dev-environment/ubuntu.md)
 * [Apple's macOS](dev-environment/macOS.md)
 
+### Virtual Machines
```

Comment:
@mlangbehn right, but the Docker lore (and support) doesn't exist yet. It seems weird to separate the sole item in a list out to its own file and creates a sort of unnecessary separation, which complicates navigating through the docs. Once the Docker lore is ready to be added, I'm all for separating the two out from this README.